MacOS/Homebrew/Install.sh

```diff
@@ -3,7 +3,13 @@
 exit_message () {
     echo ""
     echo "Oh no! Something went wrong"
-    echo "Please try to install manually or contact the Python Support Team:" 
+    echo ""
+    echo "Please visit the following web page:"
+    echo ""
+    echo "https://pythonsupport.dtu.dk/install/macos/automated-error.html"
+    open https://pythonsupport.dtu.dk/install/macos/automated-error.html
```

Comment:
could you move the `open` call to *just* before you return this, in that way we should have a bit more resilience